spec/lib/pdf_filler/az301_pdf_spec.rb

```diff
@@ -0,0 +1,33 @@
+require 'rails_helper'
+
+RSpec.describe PdfFiller::Az301Pdf do
+  include PdfSpecHelper
+
+  let(:intake) { create(:state_file_az_intake, :with_az321_contributions, :with_az322_contributions) }
+  let(:submission) { create :efile_submission, tax_return: nil, data_source: intake }
+  let(:pdf) { described_class.new(submission) }
+
+  describe '#hash_for_pdf' do
+    let(:pdf_fields) { filled_in_values(submission.reload.generate_filing_pdf.path) }
+
+    it 'uses field names that exist in the pdf' do
+      missing_fields = pdf.hash_for_pdf.keys.map(&:to_s) - pdf_fields.keys
+      expect(missing_fields).to eq([])
+    end
+
+    context "pulling fields from xml" do
+      let(:intake) { create(:state_file_az_intake, :with_az321_contributions, :with_az322_contributions) }
+
+      it "sets other fields to the correct values" do
```

Comment:
i see there are 19 lines filled in the pdf filler but only 8 here. is there a reason for not testing the other 11?